docs/api/index.rst

```diff
@@ -1,16 +1,64 @@
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/buckets/(bucket_id)/collections/(collection_id) <collection-get>`                     | :ref:`Retreive an existing collection <collection-get>` |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `DELETE` | :ref:`/buckets/(bucket_id)/collections/(collection_id) <collection-delete>`                  | :ref:`Delete a collection <collection-delete>`          |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `POST`   | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records <records-post>`               | :ref:`Upload a record <records-post>`                   |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `PUT`    | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <record-put>`     | :ref:`Replace a record <record-put>`                    |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `PATCH`  | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <record-patch>`   | :ref:`Update a record <record-patch>`                   |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records <records-get>`                | :ref:`Retrieve stored records <records-get>`            |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <records-get>`    | :ref:`Retrieve a single record <records-get>`           |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `DELETE` | :ref:`/buckets/(bucket_id)/collections/(collection_id)/records/(record_id) <record-delete>`  | :ref:`Delete a single record <record-delete>`           |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+
+Many of the listed endpoints are resource endpoints, which an be filtered,
+paginated and interracted as described in :ref:`resource-endpoints`.
```

Comment:
I was thinking about that, but actually it would mean to put the symbol next to almost all resources. Maybe it's still worth it.